apps/fabric-website/gulpfile.js

```diff
@@ -1,73 +0,0 @@
-'use strict';
-
-let build = require('@microsoft/web-library-build');
-let gulp = require('gulp');
```

Comment:
seems like there are some unrelated changes in this PR?

Reply:
just deleting the old gulp files while im here. Also included some missing typings.

Reply:
ok, just wanted to make sure it was unrelated to the PR